commons/src/main/java/com/simpletools/calculator/commons/helpers/CalculatorImpl.kt

```diff
@@ -58,21 +67,31 @@ class CalculatorImpl(calculator: Calculator) {
     }
 
     fun handleEquals() {
-        val operation = OperationFactory.forId(operator!!, firstNumber, secondNumber)
-
-        if (operation != null && (digits > 0 || operation is UnaryOperation)) {
-            setAllClear()
-            resetValues()
-            firstNumber = operation.getResult()
-            setValue(Formatter.doubleToString(firstNumber))
-            setFormula(operation.getFormula())
-            lastIsOperation = false
+        val operation : Operation?
```

Comment:
Why does this variable need to be immutable ?

Reply:
it's a const within the function. doesn't need to be `var`

Reply:
This is a safe call operator. 'OperationFactory.forId()' return a safe Operation so this variable also needs to be declared safe.